docs/source/contact/contact.rst

```diff
@@ -0,0 +1,83 @@
+Contact Us
+======================
+
+:doc:`Game Forum Q&A </game_specific_resources/ftcqa/ftcqa>`
+--------------------------------------------------------------
+
+   Anyone may view questions and answers within the *FIRST\ ®* Tech
+   Challenge game Q&A forum without a password. To submit a new
+   question, you must have a unique Q&A system user name and password
+   for your team.
+
+Volunteer Forum
+---------------
+
+   Volunteers can request access to role specific volunteer forums by
+   emailing FTCTrainingSupport@firstinspires.org. You will receive
+   access to the forum thread specific to your role.
+
+:doc:`FIRST Tech Challenge Game Manuals </manuals/game_manuals/game_manuals>`
+------------------------------------------------------------------------------
+
+
+FIRST Headquarters Pre-Event Support
+------------------------------------
+
+   | Phone: 603-666-3906
+   | Mon – Fri ~ 8:30am – 5:00pm
+   |
+   | `Firsttechchallenge@firstinspires.org <mailto:FTCTeams@firstinspires.org>`__
+
+FIRST Tech Challenge Event On-Call Support
+------------------------------------------
+
+The on call event support number is available for event personnel only.
+Please **do not** call these numbers if you are a team looking for a
+ruling, a decision, or assistance. We trust that you will not misuse
+this resource\ *.*
+
+Day of event robot control system and scoring system support:
+603-206-2450
+
+All other day of event support: 603-206-2412
+
+FIRST Websites
+--------------
+
+   `FIRST Homepage <http://www.firstinspires.org>`_ –
+
+   `FIRST Tech Challenge
+   Page <http://www.firstinspires.org/robotics/ftc>`__ – For everything
+   *FIRST* Tech Challenge.
+
+   `FIRST Tech Challenge Volunteer
+   Resources <http://www.firstinspires.org/node/5146>`__ – To access
+   public volunteer manuals.
+
+   `FIRST Tech Challenge Event
+   Schedule <http://www.firstinspires.org/team-event-search>`__ – Find
+   *FIRST* Tech Challenge events in your area.
+
+FIRST Tech Challenge Social Media
+---------------------------------
+
+   `FIRST Tech Challenge Twitter Feed <https://twitter.com/FTCTeams>`__
+   - If you are on Twitter, follow the *FIRST* Tech Challenge Twitter
+   feed for news updates.
+
+   `FIRST Tech Challenge Facebook
+   page <https://www.facebook.com/FTCTeams>`__ - If you are on Facebook,
+   follow the *FIRST* Tech Challenge page for news updates.
+
+   `FIRST Tech Challenge YouTube
+   Channel <https://www.youtube.com/user/FIRSTTechChallenge>`__ –
+   Contains training videos, game animations, news clips, and more.
+
+   `FIRST Tech Challenge
+   Blog <http://firsttechchallenge.blogspot.com/>`__ – Weekly articles
+   for the *FIRST* Tech Challenge community, including outstanding
+   volunteer recognition!
+
+   `FIRST Tech Challenge Team Email
+   Blasts <http://www.firstinspires.org/node/4311>`__ – contain the most
+   recent *FIRST* Tech Challenge news for teams.
```

Comment:
The blog and newsletter are consumption resources, not contact resources.  They'd be better referenced in a media sidebar category?  Would that media sidebar perhaps also contain the Twitter/Faceback/YouTube references?

Reply:
> The blog and newsletter are consumption resources, not contact resources. 
> They'd be better referenced in a media sidebar category? Would that media 
> sidebar perhaps also contain the Twitter/Faceback/YouTube references?

I don't like the term "media", but maybe I can warm up to this idea. Maybe a section called "Blogs / Blasts / Socials" that has this information. So our menu would look like this:

![image](https://user-images.githubusercontent.com/5846611/211852237-da2773f6-f6ef-4fa0-a3a1-67eff8ebd708.png)

-Danny